src/GeekLearning.Storage.Azure/AzureStore.cs

```diff
@@ -113,7 +78,7 @@ public AzureStore(string storeName, string connectionString, string containerNam
             blockBlob.Properties.ContentType = contentType;
             blockBlob.Properties.CacheControl = "max-age=300, must-revalidate";
             await blockBlob.SetPropertiesAsync();
-            return new Internal.AzureFileReference(blockBlob);
+            return new Internal.AzureFileReference(blockBlob, withMetadata: true);
```

Comment:
In fact, it is not the loading that we specify here, but the fact that the properties and metadata are loaded :)